DMTN-222.tex

```diff
@@ -454,7 +454,302 @@ \section{Batch Process Submission (BPS) Templates}
 \section{Full Calibration Example}
 \label{sec:example}
 
-TODO: DM-48520
+This section contains the commands used to generate, certify, chain, and verify calibrations for LSSTComCam DP1 processing.
+This is adapted from the script attached to DM-48520.
+
+Note that this is not a complete construction of all calibrations; for this we used pre-existing linearity, ptc, and bfk.
+
+In addition to the calibration building, this includes some checkpoints to download mosaic images for visual spot-checking to ensure that nothing has gone terribly wrong before continuing on all the way to verification reports.
+
```

Comment:
Unfortunately, LaTeX's verbatim is very literal, and can't understand line wrap.  I think the solution is to make a note here that the following commands have been manually line wrapped, and so lines that end with a `\` should have the following lines (which are indented for legibility) added to them.  And then to go through and manually wrap all the lines that are too long in the render.
We can probably ease this a bit by wrapping the verbatim in a `\begin{small}` block, so fewer lines need to be wrapped.  Scrolling through the render, it looks like most of the commands have been wrapped already, so it's just the `export`s and the `butler retrieve-artifacts`.